pdchaosazure/vmss/fetcher.py

```diff
@@ -57,12 +58,18 @@ def fetch_vmss(filter, configuration, secrets) -> List[dict]:
     return vmss
 
 
-#############################################################################
-# Private helper functions
-#############################################################################
-def __fetch_vmss_instances(choice, configuration, secrets) -> List[Dict]:
+def fetch_all_vmss_instances(choice, configuration, secrets) -> List[Dict]:
     vmss_instances = []
-    client = init_client(secrets, configuration)
+    secrets1 = load_secrets(secrets)
+    configuration1 = load_subscription_id(configuration)
+    with auth(secrets1) as authentication:
+        base_url = secrets1.get('cloud').endpoints.resource_manager
+        client1 = ComputeManagementClient(
+            credentials=authentication,
+            subscription_id=configuration1.get('subscription_id'),
+            base_url=base_url)
+
+        client = client1
```

Comment:
We do it exactly in init_client. We do init client as well in the top level functions. I think this is redundant. Let's pass the client to helper functions instead of creating new one

Reply:
done